dso-l2/src/main/java/com/tc/net/groups/TCGroupManagerImpl.java

```diff
@@ -515,6 +520,11 @@ public void sendTo(NodeID node, AbstractGroupMessage msg) throws GroupException
     internalSendTo(node, msg, sentCallback);
   }
 
+  @Override
+  public void sendTo(Set<String> nodes, AbstractGroupMessage msg) {
+    sendAll(msg, members.keySet().stream().filter(id -> nodes.contains(id.getName())).collect(toSet()));
+  }
+
```

Comment:
Why are these new filtered sendAll's being added?  is there lag in updating the member set?

Reply:
This change helps in sending election messages to locked members as topology might have changed while the election is running and a new server got added to `members`.